github_api_helper.js

```diff
@@ -31,10 +31,11 @@ function inProgressAPIresponse (context, headSha) {
  * Get Precaution config file contents as raw data,
  * if that file exists.
  * @param {import('probot').Context} context Probot context
+ * @param {String} customConfigPath Optional (default value config.configFilePath)
  * @return {Promise<any>} GitHub response
  * See https://developer.github.com/v3/repos/contents/#get-contents
  */
-async function getConfigFile (context) {
+async function getConfigFile (context, customConfigPath = config.configFilePath) {
```

Comment:
It's strange that we introduce a new function in commit one of this PR then immediately change it in commit two of the same PR. That's a perfectly natural way for code to evolve during development but not necessary to expose in the revision control history.

Reply:
This is a change was introduced when we added our unit tests and that's why I change it in the commit where I added the unit tests.